deploy/k8s/hstream-server.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: hstream-server-service
+  name: hstream-server
 spec:
+  type: ClusterIP
+  clusterIP: None
```

Comment:
It looks like the helm template is missing this part of the change, is this intentional?